performance_tests/consolidate_results.py

```diff
@@ -0,0 +1,48 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Script for consolidating results for execution times"""
+
+import argparse
+import csv
+
+import numpy as np
+
+
+def calculate_statistics(_csv_file):
+    """Calculate average, trimmed average, minimum, maximum and p99 statistics for execution times in a CSV file."""
+    with open(_csv_file, 'r', encoding='utf-8') as file:
+        reader = csv.reader(file)
+        data = [float(row[0]) for row in reader]
+
+    # Calculate statistics
+    if data:
```

Comment:
(nit) I don't know if you need to refactor this, but for future reference, consider using a more robust structure so we don't need to worry about the order of the returned values. If this were to change (reorder returned values, add new returned values), consumers wouldn't know that it changed. They might be grabbing `average = statistics[1]`, but a more robust structure is `average = statistics["average"]` from a dictionary

Reply:
Done